devices/sunricher.js

```diff
@@ -606,7 +606,7 @@ module.exports = [
         },
     },
     {
-        fingerprint: [{modelID: 'TERNCY-DC01', manufacturer: 'Sunricher'}],
+        fingerprint: [{modelID: 'TERNCY-DC01', manufacturerName: 'Sunricher'}],
```

Comment:
@asgothian `manufacturer` had to be `manufacturerName` , @DubhAd can you check if this works now?

Reply:
I've just tested and it works now - my Terncy devices are still detected as such and work again.